helpers/constants/constants.go

```diff
@@ -0,0 +1,20 @@
+package constants
```

Comment:
I'm not very familiar with open-source Go but it doesn't look like using many nested packages is common. To keep things lightweight, I would recommend keeping the hierarchy as flat as possible, possibly even in the same folder (from taking a quick look at popular packages, many have 20+ files in a given folder, e.g. [`pg`](https://github.com/go-pg/pg)).

Additionally, I would recommend introducing internal packages only if they have a thinner public API than their implementation. When this isn't the case (e.g. here, where everything is public), you don't benefit from the package boundary. Having too many distinct packages has a cost on maintenance.